### ISSUE TITLE
About Deep Coral loss

### ISSUE THREAD

**Issue by ddrise** [jindongwang/transferlearning#33]
HI, Wang, thx for your code, it really helps me a lot.

In your Deep Coral Code(transferlearning/code/deep/DeepCoral/Coral.py), i found you use:

xm = torch.mean(source, 1, keepdim=True) - source

But in the origianl implementation, i think they use:
xm = torch.mean(source, 0, keepdim=True) - source
Is there any thing I have missed or this is a typo?
